src/DotVVM.Framework/Resources/Scripts/utils/stringHelper.ts

```diff
@@ -6,3 +6,8 @@
 
     return tokens;
 }
+
+export function join<T>(elements: T[], delimiter: string): string {
+    let unwrappedElements = elements.map(function (element) { return ko.unwrap(element); });
```

Comment:
```suggestion
    let unwrappedElements = elements.map(ko.unwrap);
```

I'd prefer inlining it into the translation, but not that it would matter that much.